README.md

```diff
@@ -116,6 +116,35 @@ dependencies {
 }
 ```
 
+### Specifying a Checker Framework version
+
+To change the version of the Checker Framework used by the plugin (by default version 2.8.0),
+you should add `checkerFramework` entries to your `dependencies` block. 
+We recommend **explicitly** specifying the latest Checker Framework version, and updating
+often. You can find the latest version of the Checker Framework 
```

Comment:
Users don't need to find the Checker Framework.  They just need to know the version number and release notes (which are at the same place, so I'm suggesting changing the text rather than the URL).